### ISSUE TITLE
Docs: Possible mistake in "Unreliable Deployment on a Cluster of Dedicated Servers"

### ISSUE THREAD

**Issue by andersjonsson** [dotnet/orleans#4692]
I tried to follow the guide for DevelopmentClustering
https://github.com/dotnet/orleans/blob/gh-pages/src/Documentation/Deployment-and-Operations/Configuration-Guide/Typical-Configurations.md/#unreliable-deployment-on-a-cluster-of-dedicated-servers

When I ran the code the client couldn't connect, so I tried changing the client port, in the .UseStaticClustering()-call to the gateway port (30000). That seemed to do the trick.

I guess that it's a typo in the Docs but I'm extremely new to Orleans, so I don't want to assume to much.

If it is a typo I can fix it and submit a PR

Also, the ClusterId and and ServiceId mismatch between the silo and the client config, but I caught that before I even tried to run the code.

**Comment by JillHeaden** [comment]
HELP WANTED : Can someone please confirm that the suggested change is correct? If so, I will be glad to fix the document. (Also, can someone please add the "Help Wanted" label on this issue? Thanks!) 
@jason-bragg

**Comment by ReubenBond** [comment]
This is correct - the port should be the gateway port, 30000.

**Comment by JillHeaden** [comment]
Thank you, @ReubenBond !

**Comment by JillHeaden** [comment]
To confirm, line 129:
`  .UseStaticClustering(gateways)`

... is being changed to...
`    .UseStaticClustering(gatewayPort: 30000)`

**Comment by ReubenBond** [comment]
Apologies for not commenting earlier. The change should be that 11111 is changed to 30000 on lines 123, 124, & 126.

``` diff
var gateways = new IPEndPoint[]
{
-    new IPEndPoint(PRIMARY_SILO_IP_ADDRESS, 11111),
+    new IPEndPoint(PRIMARY_SILO_IP_ADDRESS, 30000),
-    new IPEndPoint(OTHER_SILO__IP_ADDRESS_1, 11111),
+    new IPEndPoint(OTHER_SILO__IP_ADDRESS_1, 30000),
    [...]
-    new IPEndPoint(OTHER_SILO__IP_ADDRESS_N, 11111),
+    new IPEndPoint(OTHER_SILO__IP_ADDRESS_N, 30000),
};
```

**Comment by JillHeaden** [comment]
No problem. I will undo the other change (as noted in a comment on the PR) and make this change instead. 

Thanks!

**Comment by ReubenBond** [comment]
Fixed by #4948
